src/js/route.js

```diff
@@ -15,10 +12,15 @@ Router.prototype.toString = function() {
 };
 
 Router.prototype.constructDomain = function() {
-    if (this.absolute)
+    if(this.name === undefined) {
```

Comment:
I'm curious as well.  Maybe we should write a test to cover the absolute param and PR this back in.